roles/osm-vector-maps/tasks/main.yml

```diff
@@ -15,9 +15,9 @@
   include_tasks: install.yml
   when: osm_vector_maps_installed is undefined
 
-- name: Install OSM Vector Maps if 'osm_vector_maps_installed' not defined, e.g. in {{ iiab_state_file }}    # /etc/iiab/iiab_state.yml
-  include_tasks: install.yml
-  when: osm_vector_maps_install | bool and not osm_vector_maps_installed is defined
+- name: Enable/Disable/Reload NGINX for OSM, if nginx_enabled
+  include_tasks: nginx.yml
+  when: nginx_enabled | bool
```

Comment:
The when clause is undesirable, nginx,yml should deal with both enabled and disabled as needed.

Reply:
> The when clause is undesirable, nginx.yml should deal with both enabled and disabled as needed.

@jvonay this `when:` clause looks ok to me, as it's conditioned on `nginx_enabled` (as in most other Ansible roles/playbooks) rather than `osm_vector_maps_enabled`.

(Or am I missing something?)